docs/shared/tutorials/react-standalone.md

```diff
@@ -581,28 +745,23 @@ Serving your app (`nx serve`) and then navigating to `/products` should give you
 
 ![products route](/shared/images/tutorial-react-standalone/react-tutorial-products-route.png)
```

Comment:
This image shows an outdated screen. New app says `Welcome to Products` not `Welcome to ProducList`